ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -117,6 +134,43 @@ class MainService : Service() {
         Toast.makeText(this, "The MainService has been gone", Toast.LENGTH_SHORT).show()
     }
 
+    private fun copyFilesToExternalDir() {
+        val am = resources.assets
+        var files: Array<String>? = null
+
+        try {
+            files = am.list("models/")
+        } catch (e: java.lang.Exception) {
+            Log.e("TAG", "#### Failed to get asset file list")
```

Comment:
```suggestion
            Log.e(TAG, "#### Failed to get asset file list")
```

Reply:
I automatically converted the Java code and missed this part😅
Thank you!!